packages/kbn-search-connectors/lib/update_connector_index_name.ts

```diff
@@ -6,29 +6,19 @@
  * Side Public License, v 1.
  */
 
-import { WriteResponseBase } from '@elastic/elasticsearch/lib/api/types';
+import { Result } from '@elastic/elasticsearch/lib/api/types';
 import { ElasticsearchClient } from '@kbn/core/server';
-import { i18n } from '@kbn/i18n';
-import { CONNECTORS_INDEX, fetchConnectorByIndexName } from '..';
 
 export const updateConnectorIndexName = async (
   client: ElasticsearchClient,
   connectorId: string,
   indexName: string
-): Promise<WriteResponseBase> => {
-  const connectorResult = await fetchConnectorByIndexName(client, indexName);
-  if (connectorResult) {
```

Comment:
does the underlying API do this check now, or fail with a different error for this case?

Assuming we're handling it but just want to confirm since we're removing the explicit handling of this case here.